src/css/main.css

```diff
@@ -38,6 +58,9 @@ a:link, a.active:link,a:hover, a:visited {
 	font-weight: bold;
 }
 /* https://github.com/material-components/material-components-web/issues/2718 */
+.mdc-text-field:not(.mdc-text-field--disabled) .mdc-floating-label {
```

Comment:
* This should be with the other text-field attributes
* Looks like the "Search" text field hint isn't visible in night mode